client/my-sites/backup/status/index.jsx

```diff
@@ -75,6 +76,8 @@ export const RealtimeStatus = ( { selectedDate } ) => {
 				} }
 			/>
 
+			<BackupGettingStarted />
```

Comment:
Is it expected to show the card in Calypso blue (at `wordpress.com/backup/:site`)?

Reply:
I've asked to confirm that, thanks for pointing this out

Reply:
We indeed show the card on both Calypso Green and Blue - I received the answer about that one.

Reference: p6TEKc-5Yg-p2#comment-11176

Reply:
I'm updating `testing plan` as well.